Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -348,17 +351,40 @@ public class EOSSessionsManager : IEOSSubManager
         private SessionDetails JoiningSessionDetails = null;
         private ulong JoinedSessionIndex = 0;
 
+        /// <summary>
+        /// When joining a Session you did not create, the Session's name is not shared with you.
+        /// Instead the Session is given a local name, starting with this, when <see cref="GenerateJoinedSessionName(bool)"/> is called.
+        /// This local name doesn't have to follow any particular formatting, but does need to be locally unique and consistent.
```

Comment:
nit: Actually, as a matter of habit I try to avoid using contractions at all. However, sometimes it is unavoidable.

Reply:
"doesn't" easily becomes "does not". But "Session's" is a possessive, is that still a contraction?